LiskHQ/lisk-hub
# Implement editing and deleting followed account - Closes #904

### What was the problem?
The followed accounts were not editable or deletable

### How did I fix it?
Added those functionalities 

### How to test it?
Try adding, deleting and editing followed accounts and check everything is correct after refresh as well 

### Review checklist
- The PR solves #904
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
